lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -44,6 +45,8 @@ class ItemListPresenter {
     private var itemListDisplayStore: ItemListDisplayStore
     private var disposeBag = DisposeBag()
 
+    var manualSync = false
```

Comment:
I don't love this stateful way of keeping track of this piece, I'd rather see the manual sync piece be another `ItemListDisplayAction` style configuration so that we can put telemetry on it if we eventually want to do that.